.swiftlint.yml

```
@@ -11,6 +11,6 @@ disabled_rules:
   - function_parameter_count
 
 included:
-- ios
-- example/ios
-- targets
\ No newline at end of file
+- packages/react-native-device-activity/ios
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 2 but found 0

(indentation)